packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -71,31 +78,72 @@ const TextInput: typeof TextAreaBase = React.forwardRef(
       <TextAreaBase
```

Comment:
Can we make sure this gets a label (even if it's visually hidden)?

Reply:
lol no time like the present, added a visually hidden label